regolith/commands.py

```diff
@@ -180,6 +185,23 @@ def yaml_to_json(rc):
         fsclient.yaml_to_json(inp, out)
 
 
+def fs_to_mongo(rc: RunControl) -> None:
+    """Import database from filesystem to mongo db.
```

Comment:
I wonder if this is clearer if it says "Convert database collection from filesystem to mongo db" rather than import